OvernightPrintsSpecTests/FFs/Basket_Feature.feature

```diff
@@ -0,0 +1,300 @@
+	And I click Get Started button on "Postcards" Product page
+	And I click Customize button on "Postcards" Template page
+	And I click Next button on "Postcards" Builder page
+	And I click Approve button on "Postcards" Builder page
+	And I click Skip button on "Postcards" Mailing List Page
+
+	Then I wait for 50 seconds for "Approve Order Page" is load
+	When I click Continue Order button on "Postcards" Approve Order Page
+
+	Then I see Basket Page
+	And I wait for 10 seconds for "500 Double Sided Business Cards" is load
+	And I wait for 10 seconds for "1000 Double Sided" is load
+	And I see following product on the Cart
+		| Name                            | Quantity Price | Material Price | Item Subtotal |
+		| 500 Double Sided Business Cards | $39.95         | $0.00          | $27.95        |
+		| 1000 Double Sided               | $126.45        | $0.00          | $92.22        |
+	And I see Basket contains "2" elements
+
+	When I click Log out button on the Header Page
+	Then I see Main page
+	And I see that user is not logged in
+
+	When I click CART button on the Header Page
+	Then I see Basket Page
+	And I wait for 10 seconds for "500 Double Sided Business Cards" is load
+	And I wait for 10 seconds for "1000 Double Sided" is load
+	And I see following product on the Cart
+		| Name                            | Quantity Price | Material Price | Item Subtotal |
+		| 500 Double Sided Business Cards | $39.95         | $0.00          | $29.35        |
+		| 1000 Double Sided               | $126.45        | $0.00          | $98.95        |
+	And I see Basket contains "2" elements
+
+	When I remove "500 Double Sided Business Cards" item from basket
+	Then I see Basket Page
+	Then I wait for 10 seconds
+	Then I don't see following product on the Cart
+		| Name                            | Quantity Price | Material Price | Item Subtotal |
+		| 500 Double Sided Business Cards | $39.95         | $0.00          | $29.35        |
+	And I see Basket contains "1" elements
+
+	When I remove "1000 Double Sided" item from basket
+	Then I see Basket Page
+	Then I wait for 10 seconds
+	Then I don't see following product on the Cart
+		| Name                            | Quantity Price | Material Price | Item Subtotal |
+		| 500 Double Sided Business Cards | $39.95         | $0.00          | $29.35        |
+		| 1000 Double Sided               | $126.45        | $0.00          | $98.95        |
+	And I see Basket contains "0" elements
+
+	When I click Log in button on the Header Page
+	And I set following parameters on Login popup
+		| Field         | Value                   |
+		| Email Address | autestomation@gmail.com |
+		| Password      | 4sep98MPcalifUSA        |
+	And I click Log in button on Login popup
+
+	Then I see that user "Test" is logged in
+	And I see Basket Page
+	And I see notification message "Your shopping cart is empty." on the "Basket Page"
+
+Scenario: Check Basket state after decline option "Professional File Review"

```

Comment:
Comment ' I suggest you to check total price before and after declining of this option.' from Google Doc has not been fixed.